elide-contrib/elide-dynamic-config-helpers/README.md

```diff
@@ -0,0 +1,23 @@
+## Validation for Dynamic Config
+
+Validate the config files in local before deployment.
+
+To build and run:
+```text
+1. mvn clean install
+2. Look for Jar File under elide-contrib/elide-dynamic-config-helpers/target directory with name matching "elide-dynamic-config-*-jar-with-dependencies.jar"
```

Comment:
You can skip step 2 and provide the relative path (listed above) in step 3a and 3b.

Reply:
Updated.